presto-main/src/main/java/com/facebook/presto/operator/scalar/CombineHashFunction.java

```diff
@@ -25,7 +29,10 @@ private CombineHashFunction() {}
 
     @ScalarFunction(value = "combine_hash", visibility = HIDDEN)
     @SqlType(StandardTypes.BIGINT)
-    public static long getHash(@SqlType(StandardTypes.BIGINT) long previousHashValue, @SqlType(StandardTypes.BIGINT) long value)
+    @ScalarFunctionConstantStats(avgRowSize = 8, distinctValuesCount = ScalarFunctionStatsUtils.ROW_COUNT_TIMES_INV_NULL_FRACTION)
```

Comment:
If the return type is an instance of `FixedWidthType` do we really need to define `avgRowSize`? I feel like it should be able to be determined without this additional metadata